docs/application/toc_all.md

```diff
@@ -859,6 +859,7 @@
 ##### [Face Detection, Recognition, and Tracking](/application/native/guides/multimedia/face-detection.md)
```

Comment:
line 858,
@nawab-samsung, didn't you find out that overview for "Visual Detection and Recognition" is missing with [this comment](https://github.com/Samsung/tizen-docs/pull/1244#discussion_r490043969)?
Are you going to skip the fix for it?

\#### Visual Detection and Recognition
\##### \[Overview](/application/native/guides/multimedia/media-vision.md)
\##### \[Barcode Detection and Generation](/application/native/guides/multimedia/image-barcode.md)

Reply:
@mijin85cho  thanks, will verify and let you know about it. 
Thanks

Reply:
I think there is no overview for it, I am facing problem in viewing this PR. May be GitHub is down. Somehow could view this.
![IMG_20201006_145302](https://user-images.githubusercontent.com/43839737/95188067-94ca6a80-07e9-11eb-83c5-8c1931072ad6.jpg)

Reply:
@mijin85cho 

We already had 

![cc12](https://user-images.githubusercontent.com/43839737/95189995-479bc800-07ec-11eb-813b-15e0a71ad293.PNG)


So I kept it as is, and just suggested to added a child to this. 
Pose detection.

Reply:
@nawab-samsung I think you confused **Visual Detection and Recognition** and **Multimedia**.
Please check that **Multimedia** has overview, but  **Visual Detection and Recognition** doesn't, in your image.

According to the toc_all.md file, the pose detection is the child page of **Visual Detection and Recognition**, so the link and description should be added to the overview of **Visual Detection and Recognition**, which doesn't listed up in the menu now.

Also I think you can see https://docs1.stg.tizen.org/staging/1244/application/native/guides/multimedia/media-vision/ page, which is result of media-vision.md in this PR update, is dangling and not included in the left menu bar, because media-vision.md is not listed in the toc_all.md.

Reply:
@mijin85cho  thanks, the change is not done in this PR as the screenshot is from live website. But yes we need to fix this issue. 
Thanks again. 

@tyoungroy  could you please add the file `/application/native/guides/multimedia/media-vision.md` to the Toc as this is the overview file. 
Your Toc will look like this after addition. Kindly verify. 


#### Visual Detection and Recognition
##### [Overview](/application/native/guides/multimedia/media-vision.md)
##### [Barcode Detection and Generation](/application/native/guides/multimedia/image-barcode.md)
##### [Face Detection, Recognition, and Tracking](/application/native/guides/multimedia/face-detection.md)
##### [Image Recognition and Tracking](/application/native/guides/multimedia/image-recognition.md)
##### [Image Classification](/application/native/guides/multimedia/image-classification.md)
##### [Pose Detection](/application/native/guides/multimedia/pose-detection.md)



**Current ToC:** 

#### Visual Detection and Recognition
##### [Barcode Detection and Generation](/application/native/guides/multimedia/image-barcode.md)
##### [Face Detection, Recognition, and Tracking](/application/native/guides/multimedia/face-detection.md)
##### [Image Recognition and Tracking](/application/native/guides/multimedia/image-recognition.md)
##### [Image Classification](/application/native/guides/multimedia/image-classification.md)
 ##### [Pose Detection](/application/native/guides/multimedia/pose-detection.md)